spec/io/io_spec.rb

```diff
@@ -291,6 +291,79 @@
         end
       end
     end
+
+    context "#from_html" do
+      let(:macau) { "file://#{Dir.pwd}/spec/fixtures/macau.html" }
+      let(:valid_markup) { "file://#{Dir.pwd}/spec/fixtures/valid_markup.html" }
+      let(:wiki_climate) { "file://#{Dir.pwd}/spec/fixtures/wiki_climate.html" }
+      let(:wiki_table_info) { "file://#{Dir.pwd}/spec/fixtures/wiki_table_info.html" }
+
+      it "returns default dataframe list from html tables" do
+        df = Daru::DataFrame.from_html(wiki_table_info)[0]
```

Comment:
This should be `subject`, and different df examples should be different `context`s